src/test/java/com/synopsys/integration/blackduck/service/dataservice/UserServiceTestIT.java

```diff
@@ -30,123 +34,158 @@
 
+    private final BlackDuckServicesFactory BLACKDUCK_SERVICES_FACTORY = UserServiceTestIT.INT_HTTP_CLIENT_TEST_HELPER.createBlackDuckServicesFactory();
+    private final BlackDuckApiClient BLACKDUCK_API_CLIENT = BLACKDUCK_SERVICES_FACTORY.getBlackDuckApiClient();
+    private final ProjectService PROJECT_SERVICE = BLACKDUCK_SERVICES_FACTORY.createProjectService();
+    private final UserGroupService USER_GROUP_SERVICE = BLACKDUCK_SERVICES_FACTORY.createUserGroupService();
+    private final ProjectUsersService PROJECT_USERS_SERVICE = BLACKDUCK_SERVICES_FACTORY.createProjectUsersService();
+
+    public UserServiceTestIT() throws IntegrationException {}
+
     @Test
     public void getProjectsForUserTestIT() throws IllegalArgumentException, IntegrationException {
-        BlackDuckServicesFactory blackDuckServicesFactory = UserServiceTestIT.INT_HTTP_CLIENT_TEST_HELPER.createBlackDuckServicesFactory();
-
-        List<ProjectView> projectsForUser = blackDuckServicesFactory.createUserGroupService().getProjectsForUser(UserServiceTestIT.INT_HTTP_CLIENT_TEST_HELPER.getTestUsername());
+        List<ProjectView> projectsForUser = BLACKDUCK_SERVICES_FACTORY.createUserGroupService().getProjectsForUser(UserServiceTestIT.INT_HTTP_CLIENT_TEST_HELPER.getTestUsername());
         assertNotNull(projectsForUser);
     }
 
     @Test
     public void getRolesForUserTestIT() throws IllegalArgumentException, IntegrationException {
-        BlackDuckServicesFactory blackDuckServicesFactory = UserServiceTestIT.INT_HTTP_CLIENT_TEST_HELPER.createBlackDuckServicesFactory();
-
-        List<RoleAssignmentView> rolesForUser = blackDuckServicesFactory.createUserGroupService().getRolesForUser(UserServiceTestIT.INT_HTTP_CLIENT_TEST_HELPER.getTestUsername());
+        List<RoleAssignmentView> rolesForUser = BLACKDUCK_SERVICES_FACTORY.createUserGroupService().getRolesForUser(UserServiceTestIT.INT_HTTP_CLIENT_TEST_HELPER.getTestUsername());
         assertTrue(rolesForUser.size() > 0);
     }
 
     @Test
     public void testAddingGroupToProject() throws IntegrationException {
-        BlackDuckServicesFactory blackDuckServicesFactory = UserServiceTestIT.INT_HTTP_CLIENT_TEST_HELPER.createBlackDuckServicesFactory();
-
-        BlackDuckApiClient blackDuckApiClient = blackDuckServicesFactory.getBlackDuckApiClient();
-        ProjectService projectService = blackDuckServicesFactory.createProjectService();
-        UserGroupService userGroupService = blackDuckServicesFactory.createUserGroupService();
-        ProjectUsersService projectUsersService = blackDuckServicesFactory.createProjectUsersService();
-
         String projectName = "user-group-project" + System.currentTimeMillis();
         String userGroupName = "user-group-test" + System.currentTimeMillis();
         ProjectView projectView = null;
         UserGroupView userGroupView = null;
 
         try {
             ProjectSyncModel projectSyncModel = ProjectSyncModel.createWithDefaults(projectName, "test");
```

Comment:
going back to camelCase would remove a lot of these diffs...not a huge plus, but perhaps worth considering